### PR TITLE
PEP: pandas/core round 5 (nanops, ops, panel*)

### PR DESCRIPTION
**Pull request by rockg**

### CODE REVIEW

File: pandas/core/nanops.py [hunk 1]
```diff
@@ -400,7 +398,7 @@ def nanvar(values, axis=None, skipna=True, ddof=1):
     avg = _ensure_numeric(values.sum(axis=axis, dtype=np.float64)) / count
     if axis is not None:
         avg = np.expand_dims(avg, axis)
-    sqr = _ensure_numeric((avg - values) ** 2)
+    sqr = _ensure_numeric((avg - values)**2)
```
Comment (wesm) [inline pandas/core/nanops.py]: We don't need spaces around this operator?

**Comment by wesm** [comment]
Looks good, but:

```
$ flake8 pandas/core/nanops.py 
pandas/core/nanops.py:401:41: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:491:16: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:491:42: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:492:16: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:492:42: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:499:45: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:528:16: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:528:42: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:529:16: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:529:42: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:530:17: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:530:43: E226 missing whitespace around arithmetic operator
pandas/core/nanops.py:545:33: E226 missing whitespace around arithmetic operator

$ flake8 pandas/core/ops.py 
pandas/core/ops.py:94:41: E226 missing whitespace around arithmetic operator
```

**Comment by rockg** [comment]
Well this is fun:
```
(development)roch@roch ~/Software/pandas $ flake8 pandas/core/nanops.py
(development)roch@roch ~/Software/pandas $ flake8 pandas/core/ops.py
(development)roch@roch ~/Software/pandas $ git status
On branch pep8-round5
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	pandas/version.py

nothing added to commit but untracked files present (use "git add" to track)
(development)roch@roch ~/Software/pandas $ 
```

**Comment by jreback** [comment]
that's pretty old file. how did that slip in?

**Comment by rockg** [comment]
I was just doing that to show no local changes.  Seems in pep8 version 1.6 they no longer enforce whitespace around the operator. I'm using 1.6.2.

**Comment by wesm** [comment]
@rockg that's super fun! for the preservation of our sanity let's settle on the >= 1.6 behavior; `**` isn't all that common anyhow

**Comment by wesm** [comment]
thank you!

**Pull request closed without merging**